app/models/case_court_report_context.rb

```diff
@@ -88,6 +89,51 @@ def org_address(path_to_template)
     @volunteer.casa_org.address if @volunteer && is_default_template
   end
 
+  # Sample output
+  #
+  # expected_topics = {
+  # "Question 1" => {topic: "Question 1", details: "Details 1", answers: [
+  #   {date: "12/02/20", medium: "Type A1, Type B1", value: "Answer 1"},
+  #   {date: "12/03/20", medium: "Type A2, Type B2", value: "Answer 3"}
+  # ]},
+  # "Question 2" => {topic: "Question 2", details: "Details 2", answers: [
+  #   {date: "12/02/20", medium: "Type A1, Type B1", value: "Answer 2"},
+  #   {date: "12/04/20", medium: "Type A3, Type B3", value: "Answer 5"}
+  # ]},
+  # "Question 3" => {topic: "Question 3", details: "Details 3", answers: [
+  #   {date: "12/03/20", medium: "Type A2, Type B2", value: "No Answer Provided"},
+  #   {date: "12/04/20", medium: "Type A3, Type B3", value: "No Answer Provided"}
+  # ]}
+  # }
+  def court_topics
+    topics = ContactTopic
```

Comment:
nice query. It could be slow but we are working with small numbers I think we're fine